src/topology/algebra/group.lean

```diff
@@ -167,12 +167,13 @@ section quotient_topological_group
 variables [topological_space α] [group α] [topological_group α] (N : set α) [normal_subgroup N]
 
 @[to_additive]
-instance : topological_space (quotient_group.quotient N) :=
+instance {α : Type u} [group α] [topological_space α] (N : set α) [normal_subgroup N] :
+  topological_space (quotient_group.quotient N) :=
 by dunfold quotient_group.quotient; apply_instance
 
 open quotient_group
 @[to_additive quotient_add_group_saturate]
-lemma quotient_group_saturate (s : set α) :
+lemma quotient_group_saturate {α : Type u} [group α] (N : set α) [normal_subgroup N] (s : set α) :
```

Comment:
Idem.

Reply:
Idem, this removes `[topological_group α]`.